### ISSUE TITLE
[Feature Request] Add option to "press new hotkey" instead of writing

### ISSUE THREAD

**Issue by Coreusa** [oliverschwendener/ueli#1244]
At the moment, updating the hotkey for the app requires you to type what the hotkey is. This can lead to misspellings or invalid input from the user. It's much easier to allow the user to change hotkey by pressing the keys they want.

**Comment by RadikaRules** [comment]
Yeah, that threw me for a loop given how you want it is how it worked in version 8 IIRC.
I suppose a lot of things ([like this](https://github.com/oliverschwendener/ueli/issues/38)) are currently broken or missing because 9 appears to be a major rework,
I'm just happy Oliver can actually listen to people. If this was closed source, we'd all be SOL
